BackToOpener/explainer.md

```diff
@@ -0,0 +1,86 @@
+
+This document status: Active  
+Expected venue: [WHAT WG](https://whatwg.org/)  
+Current version: this document
+
+## Introduction
+
+A relatively modern paradigm has emerged where chat interfaces (e.g. ChatGPT, Google AI mode, Copilot, Gemini) often open content in new tabs rather than opening in the current tab. This makes sense for chat interfaces, as this approach helps the user with multitasking and preserves the conversational context. This is in addition to various search engines that already opens links in a new tab (e.g. Yahoo, Bing, Perplexity). However, this causes tab proliferation, and also prevents a natural back navigation flow. 
+
+This proposal introduces an opt‑in mechanism that signals the browser to insert the opener as an initial entry in the new tab's history. Additionally, when the user navigates back in the new tab, the UA will automatically return focus to the originating tab and close the new tab, provided the originating tab is still active. This feature enhances user experience by supporting a logical back navigation flow, and reduce tab clutter at the same time.
+
+## User-Facing Problem
+
+The new-tab default of chat interfaces and search engines can frustrate users by causing tab proliferation and making it difficult to return to the original conversation or context. Traditional [guidance](https://www.nngroup.com/articles/new-browser-windows-and-tabs/) warns against breaking the user's flow by opening new tabs. At the same time, modern chat interfaces prioritize multitasking and context preservation, which thus prefers opening in new tab.
+
+This proposal bridges the gap by creating a clear navigation pathway back to the originating tab, ensuring users can effortlessly return without navigating through tabs. It additionally helps reduce tab clutter.
+
+## Goals and Use Cases
+
+The primary goal is to allow web developers to maintain a connected navigation experience, where the source context is always within reach even when content is loaded in new tabs. Key use cases include:
+
+- Search Engines: Users can click on search results and then return to their search results page.
+- Conversational Chat Interfaces: Users can click on links and navigate back to the conversation window.
+
+## Proposed Approach
+
+Developers can signal their intent via `window.open()` and `<a>` elements. The browser will then handle the navigation logic, ensuring that when the user navigates back in the new tab, it automatically returns focus to the originating tab and closes the new tab if the opener is still active.
+
+- For `window.open()`, we propose introducing a new `windowFeatures` parameter called `addOpenerToHistory`. When this feature is specified, the browser will add the opener to the new tab's history.
```

Comment:
We should think more about the name here-- e.g. should it actually be a history entry or not?

Is there some other name that conveys more about the behavior? I can't immediately think of a better name, however.

Reply:
+1 on the naming comment. I don't necessarily think we need to bikeshed/block on that though.